doc/releases/release-notes-3.6.rst

```diff
@@ -50,13 +50,38 @@ Bluetooth
 
 * Audio
 
+  * Changed :c:struct:`bt_bap_scan_delegator_subgroup` to :c:struct:`bt_bap_bass_subgroup` and
```

Comment:
```suggestion
  * Changed struct ``bt_bap_scan_delegator_subgroup`` to :c:struct:`bt_bap_bass_subgroup` and
```

Reply:
Oh yeah, I guess the reference to the old struct will no longer work :) 

Shouldn't the documentation build have caught this an error, or does those thing not trigger errors/warnings?

Reply:
Good question, need to look into that.